app/models/dossier.rb

```diff
@@ -567,7 +567,7 @@ def can_be_deleted_by_administration?(reason)
   end
 
   def can_be_deleted_by_automatic?(reason)
-    reason == :expired && !en_instruction?
+    reason == :expired && !en_instruction? || reason == :not_modified_for_a_long_time && brouillon?
```

Comment:
d'un point de vue lisibilité surtout avec le mélange de || et && je serais plus à l'aise avec quelque chose comme

```suggestion
    return true if reason == :expired && !en_instruction?
    return true if reason == :not_modified_for_a_long_time && brouillon?
    
    false
```